### PR TITLE
refactored and cleaned up the tests

### PR DESCRIPTION
**Pull request by joshuahannan**
Utilized utility functions to the tests for:
* Constructing a transaction
* Executing a Script
* Creating a Cadence `UFix64`

### CODE REVIEW

File: lib/go/test/go.mod [hunk 1]
```diff
@@ -3,12 +3,12 @@ module github.com/onflow/lib/go/flow-core-contracts
 go 1.14
 
 require (
-	github.com/onflow/cadence v0.12.3
-	github.com/onflow/flow-core-contracts/lib/go/contracts v0.6.1-0.20210118202839-f8f9d9a04bcb
+	github.com/onflow/cadence v0.12.5
+	github.com/onflow/flow-core-contracts/lib/go/contracts v0.7.1
 	github.com/onflow/flow-core-contracts/lib/go/templates v0.0.0-00010101000000-000000000000
-	github.com/onflow/flow-emulator v0.14.0-beta3
+	github.com/onflow/flow-emulator v0.14.0
```
Comment (turbolent) [inline lib/go/test/go.mod:9]: Maybe update to [v0.14.2](https://github.com/onflow/flow-emulator/releases/tag/v0.14.2)

File: transactions/lockedTokens/user/get_total_balance.cdc [hunk 2]
```diff
@@ -37,7 +37,6 @@ pub fun main(address: Address): UFix64 {
         let nodeInfo = FlowIDTableStaking.NodeInfo(nodeID: nodeStakerRef.id)
         sum = sum 
               + nodeInfo.tokensStaked
-              + nodeInfo.totalTokensStaked
```
Comment (turbolent) [inline transactions/lockedTokens/user/get_total_balance.cdc:40]: Why was this removed?

File: transactions/lockedTokens/user/get_total_balance.cdc [hunk 3]
```diff
@@ -78,7 +77,6 @@ pub fun main(address: Address): UFix64 {
             let nodeInfo = FlowIDTableStaking.NodeInfo(nodeID: nodeID)
             sum = sum
                   + nodeInfo.tokensStaked
-                  + nodeInfo.totalTokensStaked
```
Comment (turbolent) [inline transactions/lockedTokens/user/get_total_balance.cdc:81]: Why was this removed?

**Review by turbolent — COMMENTED** [review]
Looks good, nice cleanup!

File: lib/go/test/service_test.go [hunk 4]
```diff
@@ -45,38 +45,24 @@ func TestContracts(t *testing.T) {
...
 
-	result, err = b.ExecuteScript(templates.GenerateGetStorageFeeMinimumScript(env), nil)
-	require.NoError(t, err)
-	if !assert.True(t, result.Succeeded()) {
-		t.Log(result.Error.Error())
-	}
-	min := result.Value
-	assertEqual(t, CadenceUFix64("0.0"), min.(cadence.UFix64))
+		result := executeScriptAndCheck(t, b, templates.GenerateGetStorageFeeConversionScript(env), nil)
+		assertEqual(t, CadenceUFix64("1.0"), result.(cadence.UFix64))
```
Comment (turbolent) [inline lib/go/test/service_test.go:51]: Try to remove the casts in equality assertions, i.e. just write
```suggestion
		assertEqual(t, CadenceUFix64("1.0"), result)
```

If the result is not a `cadence.UFix64`, the current assertion will crash the test.
The assertion already checks the types match.

File: transactions/lockedTokens/user/get_total_balance.cdc (hunk 2 above)
Comment (joshuahannan) [reply transactions/lockedTokens/user/get_total_balance.cdc:40]: That is a derived field so it doesn't actually reflect the total balance. I saw that is shouldn't be there while doing these and thought I would just get rid of it in the same PR

File: transactions/lockedTokens/user/get_total_balance.cdc (hunk 3 above)
Comment (joshuahannan) [reply transactions/lockedTokens/user/get_total_balance.cdc:81]: Same as above

**Comment by joshuahannan** [comment]
@turbolent I addressed your comments in 7e60b18. Does it look good now?

**Review by turbolent — APPROVED** [review]
Nice cleanup! 😍
